### PR TITLE
VZ-5698 velero private registry

### PR DESCRIPTION
**Pull request by theabmitra**
Provide a summary of the change and which issue (i.e. ticket) is fixed.
If there are any dependencies, for example on a PR in another repository, list those.

### CODE REVIEW

File: ci/private-registry/Jenkinsfile [hunk 1]
```diff
@@ -81,7 +81,7 @@ pipeline {
         OCR_CREDS = credentials('ocr-pull-and-push-account')
         OCR_REPO = 'container-registry.oracle.com'
         IMAGE_PULL_SECRET = 'verrazzano-container-registry'
-        INSTALL_CONFIG_FILE_KIND = "./tests/e2e/config/scripts/install-verrazzano-kind-prometheus-enabled.yaml"
+        INSTALL_CONFIG_FILE_KIND = "./tests/e2e/config/scripts/install-verrazzano-kind-prometheus-velero-enabled.yaml"
```
Comment (michael-cico) [inline ci/private-registry/Jenkinsfile:84]: Maybe just call this file `install-verrazzano-kind-private-registry.yaml`?

Comment (theabmitra) [reply ci/private-registry/Jenkinsfile:84]: yes we can rename the file

File: tests/e2e/verify-install/velero/velero_test.go [hunk 2]
```diff
@@ -96,8 +96,8 @@ var _ = t.Describe("Velero", Label("f:platform-lcm.install"), func() {
 						pkg.Log(pkg.Error, fmt.Sprintf("Unable to retrieve configmap %s in the namespace: %s, error: %v", veleroRestoreHelperConfigMap, constants.VeleroNameSpace, err))
 						return false
 					}
-					if !strings.HasPrefix(cfgMap.Data["image"], resticHelperImage) {
-						pkg.Log(pkg.Error, fmt.Sprintf("Configmap %s does not have the image prefix %s in the namespace: %s", veleroRestoreHelperConfigMap, resticHelperImage, constants.VeleroNameSpace))
+					if !strings.Contains(cfgMap.Data["image"], resticHelperImage) {
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply tests/e2e/verify-install/velero/velero_test.go]: Updated the logic to check for prefix

**Review by michael-cico — DISMISSED** [review]

**Review by michael-cico — APPROVED** [review]

File: tests/e2e/verify-install/velero/velero_test.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ const (
 	waitTimeout                  = 3 * time.Minute
 	pollingInterval              = 10 * time.Second
 	veleroRestoreHelperConfigMap = "restic-restore-action-config"
-	resticHelperImage            = "ghcr.io/verrazzano/velero-restic-restore-helper"
+	resticHelperImage            = "velero-restic-restore-helper"
```
[the remark this answers is not in the document]
Comment (pshirodkar-orcl) [reply tests/e2e/verify-install/velero/velero_test.go:23]: bom-validator likely should have an entry for ```velero``` namespace as well https://github.com/verrazzano/verrazzano/blob/060d07326341e12e0bb1d69d18fa51ca8d9b273a/tools/bom-validator/bom-validator.go#L80.

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**